pacfinder.go

```diff
@@ -46,7 +46,7 @@ func findPACURLForDarwin() (string, error) {
 	if err := cmd.Start(); err != nil {
 		return "", err
 	}
-	defer cmd.Wait()
+	defer cmd.Wait() // nolint:errcheck
```

Comment:
should be `//nolint:errcheck` - i.e. no space after comment token - these are comments for the machine. not humans. Hard to find definitive info about this, but https://github.com/golang/go/issues/37974#issuecomment-607510041 is a starting point.

Reply:
Done.